src/main/java/org/opentripplanner/middleware/bugsnag/jobs/BugsnagEventRequestJob.java

```diff
@@ -0,0 +1,24 @@
+package org.opentripplanner.middleware.bugsnag.jobs;
+
+import org.opentripplanner.middleware.bugsnag.BugsnagDispatcher;
+import org.opentripplanner.middleware.models.BugsnagEventRequest;
+import org.opentripplanner.middleware.persistence.Persistence;
+import org.opentripplanner.middleware.persistence.TypedPersistence;
+
+/**
+ * Job to make event data request to Bugsnag and store the initial response.
+ */
+public class BugsnagEventRequestJob implements Runnable {
```

Comment:
I don't quite understand what Bugsnag Event Requests are. Might be worth having a video chat so everyone is familiar with this.

Reply:
I've updated to include a reference and added information to the design doc:

https://docs.google.com/document/d/1idJCm4ZfLSeXyncOdeyRJB1y5zQmsdSHy4kU48qx7yY/edit#heading=h.vllffag90i4q